### PR TITLE
patch Homepage

### PR DESCRIPTION
**Pull request by alisonzthu**

### CODE REVIEW

File: client/src/reducers/index.jsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from 'redux';
+//change these two reducers to ours
+import auctionReducer from './auctionReducer.jsx';
+const todoApp = combineReducers({auctionReducer});
```
Comment (wdku5791) [inline client/src/reducers/index.jsx]: your syntax doesn't seem consistent from {auctionReducer} vs { combineReducers }

Comment (alisonzthu) [inline client/src/reducers/index.jsx]: more specific plz? like which line vs which line?

Comment (alisonzthu) [inline client/src/reducers/index.jsx]: changed accordingly. thx!

Comment (abianco3) [inline client/src/reducers/index.jsx]: nice catch winston. I'll go ahead and merge

**Pull request merged**